test/Microsoft.AspNetCore.Server.KestrelTests/HttpsConnectionFilterTests.cs

```diff
@@ -301,5 +312,59 @@ public class HttpsConnectionFilterTests
 #endif
             }
         }
+
+        // https://github.com/aspnet/KestrelHttpServer/issues/240
+        // This test currently fails on mono because of an issue with SslStream.
```

Comment:
I think we should be consistent on whether we use ConditionalFact or Fact in these tests. Can we just skip mono? I assume we aren't running tests on mono right now, but if it's not too hard it's probably better to explicitly skip it if we know it doesn't work?